distribution.md

```diff
@@ -69,3 +70,38 @@ Each stack ID MUST only be present once.
 For a given stack, the `mixins` list MUST enumerate mixins such that no included buildpacks are missing a mixin for the stack.
 
 Fewer stack entries as well as additional mixins for a stack entry MAY be specified.
+
+### Build Image
+
+The following defines how a build image SHOULD be packaged for distribution as an OCI Image. The build image is the component that provides the base image from which the build environment is constructed.
+
+The image configuration refers to the OCI Image configuration as mentioned [here](https://github.com/opencontainers/image-spec/blob/main/config.md#properties).
+
+#### Image Configuration
+
+The Build Image MUST contain the following configurations:
+
+* Image Config's `config.User` field MUST be set to a non-root user with a writable home directory.
```

Comment:
Recommendation: This MUST be a UID, not a username. Many tools can't open the image and read /etc/passwd to translate the username to a UID.